.github/workflows/slither.yml

```
@@ -0,0 +1,46 @@
+name: Slither
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - "*"
```

```suggestion
      - "**"
```
I remembered/learned this week that "*" here does not match branch names with a slash in them, but "**" does.

---

Yeah, I thought that `**` wasn't necessary for `pull_request` matching, because it actually matches against the branch you're merging _into_ (which would usually be `main`), but I suppose if you're merging into another feature branch, `*` could fail to trigger